src/platforms/kotlin-multiplatform/dsym.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Upload Debug Symbols
+sidebar_order: 1
+description: "Learn more about how to upload debug symbols in Sentry Kotlin Multiplatform."
+---
+
+To symbolicate your stacktraces, you need to provide debug information to Sentry.
+The symbolication process unscrambles the stacktraces to reveal the function, file names, and line numbers of the crash.
+
+## Apple
+
+For Apple applications please follow the iOS documentation on [Uploading Debug Symbols](/platforms/apple/guides/ios/dsym) to set up debug symbols upload.
+
+## Android
+
+### Gradle
+
+To upload ProGuard mapping files via Gradle you need to install the [Sentry Android Gradle](/platforms/android/gradle) plugin:
+
+```kotlin {filename:androidApp/build.gradle.kts}
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("com.android.application")
+    id("io.sentry.android.gradle") version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+
+// Prevent Sentry from being included in the Android app through the AGP.
+configurations {
+    compileOnly {
+        exclude(group = "io.sentry", module = "sentry")
+        exclude(group = "io.sentry", module = "sentry-android")
+    }
+}
```

Comment:
I'd rather add a plugin's code snippet for disabling the `autoInstallation.enabled` feature.

Reply:
autoInstallation disable won't exclude `io.sentry.Sentry`, maybe a mix of both? something like this:

```
configurations {
    compileOnly {
        exclude(group = "io.sentry", module = "sentry")
    }
}

sentry {
  autoInstallation {
    enabled.set(false)
  }
}
```

Reply:
it should actually, if it's not, then it's a bug most likely

Reply:
I'd add the code snippet for `autoInstallation` yes.

> autoInstallation disabled won't prevent installing io.sentry.Sentry

I don't follow that, disabling the `autoInstallation` is pretty much doing that, maybe @romtsn knows?

Reply:
I tested it out a couple times and could still use `io.sentry.Sentry` with `autoInstallation` false but not `SentryAndroid`, but I'll double check to make sure

Reply:
continued in https://github.com/getsentry/sentry-docs/pull/7024
It works as intended and only 
```
sentry {
  autoInstallation {
    enabled.set(false)
  }
}
```

is needed